pages.zh/common/seq.md

```diff
@@ -0,0 +1,20 @@
+# seq
+
+> 打印数字序列到标准输出。
+> 更多信息：<https://www.gnu.org/software/coreutils/seq>.
+
+- 从1到10的序列：
```

Comment:
Add spaces between fullwidth and halfwidth characters. ([Chinese style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#chinese-specific-rules) rule 2)
```suggestion
- 从 1 到 10 的序列：
```